apps/parsley/src/snapshot.test.ts

```diff
@@ -116,9 +127,9 @@ describe(`${options.suite}`, () => {
           const snapshotPath = path.join(
             storyDirectory,
             options.snapshotsDirName,
-            `${storyBookFileBaseName}${options.snapshotExtension}`,
+            `${storyBookFileBaseName}_${name}${options.snapshotExtension}`,
           );
-          expect(container).toMatchSpecificSnapshot(snapshotPath);
+          expect(container).toMatchFileSnapshot(snapshotPath);
```

Comment:
Based on the [documentation](https://vitest.dev/api/expect.html#tomatchfilesnapshot), it seems like we should await this:

```suggestion
          await expect(container).toMatchFileSnapshot(snapshotPath);
```